multi-agent/adapters/inbound/flask/endpoints/sessions.py

```
@@ -64,10 +69,12 @@ def _check_session_busy(session_id: str, session_type: str, svc):
 @with_require_identity_authorization
 @from_body({
     "blueprint_id": fields.Str(data_key="blueprintId", required=True),
-    "metadata": fields.Dict(data_key="metadata", required=False, load_default=lambda: {}, dump_default=lambda: {})
+    "metadata": fields.Dict(data_key="metadata", required=False, load_default=lambda: {}, dump_default=lambda: {}),
```

Ruff (0.15.20)

[warning] Prefer `dict` over useless lambda

Replace with `lambda` with `dict`

(PIE807)